pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "lido-oracle"
+version = "2.1.1"
+description = "Oracle daemon for Lido decentralized staking service. Collects and reports Ethereum 2.0 beacon chain states (the number of visible validators and their summarized balances) to the Lido dApp contract running on Ethereum 1.0 side."
+authors = ["Raman Siamionau <raman.s@lido.fi>"]
```

Comment:
Could you please also mention folks who were involved in development before the oracle was transferred to the tooling team for support?

Reply:
Oh, generated automatically. I removed myself from list. 
Added top 3 developers to list, but they don't work in Lido now, so I just added their names.